packages/api-client/src/views/Request/libs/oauth2.ts

```diff
@@ -16,45 +17,108 @@ export const isOauth2Example = (
 ): example is SecuritySchemeOauth2ExampleValue =>
   example.type.startsWith('oauth')
 
+type PKCEState = {
+  codeVerifier: string
+  codeChallenge: string
+  codeChallengeMethod: string
```

Comment:
According to proto there is 'plain' and 'S256'. Make 'S256' default if not explicitly provided  plain. Plain is discouraged but some providers don't support S256, so make it option.